### PR TITLE
Adds automatic and manual refresh

### PR DESCRIPTION
**Pull request by DylanVerstraete**
- Page will now refresh every 10 minutes of the user stays on the page
- Page will refresh is the user has not visited the page since 10 minutes
- Button for manual refresh
- Refactored code for reusability

### CODE REVIEW

File: tools/explorer/frontend/src/App.vue [hunk 1]
```diff
@@ -93,8 +101,8 @@
 </template>
 
 <script>
-import { mapGetters } from 'vuex'
-
+import { mapGetters, mapActions } from 'vuex'
+/* eslint-disable */
```
Comment (zaibon) [inline tools/explorer/frontend/src/App.vue]: is this needed ?

**Review by zaibon — APPROVED** [review]
very cool :-)

File: tools/explorer/frontend/src/App.vue (hunk 1 above)
Comment (DylanVerstraete) [reply tools/explorer/frontend/src/App.vue]: ah no, oops! :D

**Comment by zaibon** [comment]
moved to https://github.com/threefoldtech/tfexplorer/pull/3

**Pull request closed without merging**